src/content/translations/de/enterprise/index.md

```diff
@@ -76,10 +76,10 @@ Verschiedene Organisationen unternahmen gemeinsame Anstrengungen, um Ethereum un
 
 Die Erstellung einer Anwendung auf einer Layer-2-Skalierbarkeitslösung [kann helfen, viele der Bedenken, die Unternehmen früher dazu gebracht haben, auf privaten Blockchains aufzubauen, zu beseitigen](https://entethalliance.org/how-ethereum-layer-2-scaling-solutions-address-barriers-to-enterprises-building-on-mainnet/), behält jedoch die Vorteile des Aufbaus auf dem Mainnet.
 
-Beispiele für L2-Lösungen, die produktionsbereit sind oder bald sein werden:
+Beispiele für L2-Lösungen, die produktionsbereit sind oder es bald sein werden:
 
 - Optimistic Rollups (Daten on-chain, Betrugsnachweise)
-  - [Optimism](https://optimism.io/)
+  - [Optimismus](https://optimism.io/)
```

Comment:
```suggestion
  - [Optimism](https://optimism.io/)
```

I dont think this is supposed to be translated

Reply:
Exactly, good catch @corwintines.